python/whylogs/experimental/core/metrics/embedding_metric.py

```diff
@@ -166,10 +167,26 @@ def _update_submetrics(self, submetric: str, data: PreprocessedColumn) -> None:
             self.submetrics[submetric][key].columnar_update(data)
 
     def columnar_update(self, data: PreprocessedColumn) -> OperationResult:
-        if data.numpy.ints is None and data.numpy.floats is None:
+        if data.numpy.ints is None and data.numpy.floats is None and data.pandas.vectors is None:
```

Comment:
I think we want to check the `ListView` and `PandasView` vectors now, and _not_ the `NumpyView` anymore.

Reply:
This was done now.